pkg/generators/markers.go

```diff
@@ -73,8 +85,36 @@ type CommentTags struct {
 	// Default  any  `json:"default,omitempty"`
 }
 
+// Returns the schema for the given CommentTags instance.
+// This is the final authoritative schema for the comment tags
+func (c commentTags) ValidationSchema() (*spec.Schema, error) {
+	res := spec.Schema{
+		SchemaProps: c.SchemaProps,
+	}
+
+	if res.AllOf != nil {
```

Comment:
What is this line for? Isn't AllOf already part of the schemaprops?

Reply:
remnant from a planned change. removed